src/app/games/game-list/game-list.component.html

```diff
@@ -32,9 +32,9 @@
         [tooltip]="game.state"
       ></div>
     </div>
-    <div *ngIf="game.score !== undefined"  class="game-list-item__score_block">
-      <div class="red">{{ game.score.red }}</div>
+    <div *ngIf="game.score !== undefined" class="game-list-item__score-block">
       <div class="blu">{{ game.score.blu }}</div>
+      <div class="red">{{ game.score.red }}</div>
```

Comment:
What was the reason behind the decision to reorder the colors?

Reply:
On the game page BLU is on the left, thus being logically before RED; I wanted to keep this order here.